RecoLocalTracker/SiPixelRecHits/plugins/gpuPixelRecHits.h

```diff
@@ -107,12 +105,10 @@ namespace gpuPixelRecHits {
         clusParams.Q_l_Y[ic] = 0;
       }
 
-      first += threadIdx.x;
-
       __syncthreads();
 
-      // one thead per "digi"
-
+      // one thread per "digi"
+      auto first = clusters.moduleStart(1 + blockIdx.x) + threadIdx.x;
```

Comment:
and NOW it is after __syncthreads();!!!

Reply:
ok, irrelevant: first is thread local

Reply:
yes - I just moved it to the first place where it was used